### PR TITLE
THREESCALE-10719  Community Release Procedure for 3scale Operator

### PR DESCRIPTION
**Pull request by valerymo**
Jira: https://issues.redhat.com/browse/THREESCALE-10719

3scale-operator Community Release Process Doc

### CODE REVIEW

File: doc/community-release-build.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+[Introduction](#introduction)  
+[Main Steps Briefly](#main-steps-briefly)  
+[Versioning Strategy](#versioning-strategy)  
+[Steps to Perform Community Release in 3scale-operator repo](#steps-to-perform-community-release-in-3scale-operator-repo)  
+[Prepare release in community-operators-prod repo](#prepare-release-in-community-operators-prod-repo)
+[References](#references)
+
+
+## Introduction
+* Community Release created in parallel with Minor Product Releases or patch releases, and not for Major release.
```
Comment (briangallagher) [inline doc/community-release-build.md:12]: @valerymo  not for Major - is this correct. 
There will most likely never be another major 3scale release but if there was I think we would also release major community

File: doc/community-release-build.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+  * Community Release is not required for every productized build or every product patch release.
+  * In this document: Product Releases == Product Releases or patch releases
```
Comment (briangallagher) [inline doc/community-release-build.md:14]: @valerymo Don't think this line is required. Product patch release is still a product release? Do we need to make the distinction?

File: doc/community-release-build.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+- Tag will be applied to the Community Release Branch
+- Tag will be applied after Testing completion.
+
+```
+$ cd 3scale-operator
+$ git tag -a v0.11.0  -m "Community Release branch 3scale-community-v0.11.0, based on product 3scale-2.14.1-GA" ab1783b4207e43480bf7538d62ecbd83636ecf0c
+$ git push myfork v0.11.0
+```
+
+* Compare Product and Community release tags
```
Comment (briangallagher) [inline doc/community-release-build.md:107]: @valerymo Is it worth adding something here to explain why we are doing the comparison and what actions might be taken based on the results of the comparison

File: doc/community-release-build.md (hunk 1 above)
Comment (valerymo) [reply doc/community-release-build.md]: @briangallagher  Thank you for comments. Updated, separate commit. Hope now it looks better. Thanks

File: doc/community-release-build.md (hunk 2 above)
Comment (valerymo) [reply doc/community-release-build.md]: removed. Thank you

File: doc/community-release-build.md (hunk 3 above)
Comment (valerymo) [reply doc/community-release-build.md]: Thank you for comment. I looked again now, and seems to me it's not needed; removed.

File: doc/community-release-build.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+```
+community-operators-prod/operators/3scale-community-operator/0.11.0
+```
+
+* Copy manifests from 3scale-operator Community release to community-operators-prod, as for example From  [3scale-operator PR](https://github.com/3scale/3scale-operator/pull/950) To [community-operators-prod/operators/3scale-community-operator/0.11.0](community-operators-prod/operators/3scale-community-operator/0.11.0)
+
+* Update CSV. These are things that need pay attention:
+  * Components images
+  * Databases images
+  *  CVS version
```
Comment (MStokluska) [inline doc/community-release-build.md:141]:
```suggestion
  *  CSV version
```

**Pull request merged**